lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -1,138 +1,232 @@
 <div class="adf-metadata-properties">
-    <mat-accordion displayMode="flat"
-                   [multi]="multi">
-        <mat-expansion-panel *ngIf="displayDefaultProperties"
-                             [expanded]="canExpandProperties()"
-                             [attr.data-automation-id]="'adf-metadata-group-properties'">
+    <mat-accordion displayMode="flat" [multi]="multi">
+        <mat-expansion-panel
+            *ngIf="displayDefaultProperties"
+            [expanded]="isGeneralPanelExpanded"
```

Comment:
We have 3 related lines: 
```html
[expanded]="isGeneralPanelExpanded"
(opened)="isGeneralPanelExpanded = true"
(closed)="isGeneralPanelExpanded = false"
```

expanded is two-way binding so we can do probably just

 [(expanded)]="isGeneralPanelExpanded"

and remove events for opened and closed as no need to handle it separately

Reply:
good point